### DISCUSSION TITLE
How to convert units into VW?

### DISCUSSION THREAD

**Discussion by jincandev** [tailwindlabs/tailwindcss#8891] [Help]
My project is packaged with `webpack5`, and the unit of conversion using the `postcss-px-to-viewport-8-plugin` plug-in is `vw`. How to convert the unit of tailwind into `vw`??

**Comment by arisac** [comment]
took a look, interesting plugin, don't know how people will actually use it.
most size units in tailwindcss are `rem` and `px`, but you can also use `vw` in tailwindcss.
e.g. `w-full` is `width: 100vw` ref: https://tailwindcss.com/docs/width

if you want to use `vw` as unit in your projects, you can always use arbitrary values like `w-[5vw]`
e.g. https://play.tailwindcss.com/A5CEXMmaRQ

**Reply by wangrongding** [reply]
Is there a better solution or plugin for converting px in w-[50px] to rem or vw?

![image](https://user-images.githubusercontent.com/42437658/202641356-0f2b9f34-5523-48d2-aa1a-295770437dac.png)


I'm using vue3 with the postcss-px-to-viewport library, but it can't handle w-[50px] between lines, only properties with px in the style tag🥲
